### ISSUE TITLE
Error when no bit_rate present on format info

### ISSUE THREAD

**Issue by gdavila** [itu-p1203/itu-p1203#21]
I have noticed that for some reason, some video samples don't have the `bit_rate` field  in its video container.
~~~ 
ffprobe  video\=952000.track_id\=10010.mp4 -show_format -of json
{
    "format": {
        "filename": "video=952000.track_id=10010.mp4",
        "nb_streams": 1,
        "nb_programs": 0,
        "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
        "format_long_name": "QuickTime / MOV",
        "start_time": "165704726.034667",
        "duration": "165704864.105933",
        "size": "16021510",
        "probe_score": 100,
        "tags": {
            "major_brand": "iso6",
            "minor_version": "0",
            "compatible_brands": "iso6dash"
        }
    }
}
~~~ 

When this happen, the python script doesn't work:
~~~ 
p1203-standalone video\=952000.track_id\=10010.mp4 -m 3 --only-pv --debug 
DEBUG - __main__ - Running extract_from_segment_files to get input report: video=952000.track_id=10010.mp4 mode 3
ERROR - __main__ - Error during scriptprocessing, exiting: 'KeyError' object has no attribute 'output'
~~~ 

As far as I understood, the container `bit_rate` info is not really used. So it could be removed or an error handling could be added to deal with this behavior on [extractor.py#L410](https://github.com/itu-p1203/itu-p1203/blob/master/itu_p1203/extractor.py#L410), i. e., 
~~~py
        try:
        	info["bit_rate"] = int(info["bit_rate"])
        except KeyError:
        	info["bit_rate"] = 0
~~~ 

btw, it takes me a while to find the error. This could be easily improved by adding additional log-level to the loggers that handle the exceptions on the __main__.py  :) 

i. .e., 
~~~py
logger.error("Error during processing, exiting: {}".format(e), exc_info=True)
~~~

**Comment by slhck** [comment]
Thanks for your suggestion! You are right about both aspects, seems that this value for the bitrate is not needed, and yes, the error logging could be better.  Would you be willing to provide a PR including these two fixes?

**Comment by slhck** [comment]
Closed via #22
